mlflow/server/js/src/components/artifact-view-components/ShowArtifactPage.js

```diff
@@ -15,7 +15,7 @@ class ShowArtifactPage extends Component {
 
   render() {
     if (this.props.path) {
-      if (IMAGE_EXTENSIONS.has(getExtension(this.props.path))) {
+      if (IMAGE_EXTENSIONS.has(getExtension(this.props.path).toLowerCase())) {
```

Comment:
Is `getExtension(this.props.path)` guarantteed to be non-Null?

Reply:
Addressed!